microk8s-resources/actions/kube-api-audit.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: audit.k8s.io/v1 # This is required.
```

Comment:
Do not place yaml files under actions because they are considered addons. In this case for example you have created a `microk8s enable kube-api-audit` addon.

What you should do is to create a falco directory and put all your yamls inside there.